modules/rostests/apitests/kernel32/InitOnce.c

```diff
@@ -169,6 +170,16 @@ START_TEST(InitOnce)
         goto _test_async;
     }
 
+    /* Call again to check whether initialization has completed */
+    fPending = 0xdeadbeef;
+    bRet = pfnInitOnceBeginInitialize(&InitOnce,
+                                      INIT_ONCE_CHECK_ONLY,
+                                      &fPending,
+                                      (LPVOID*)&ulContextData);
+    ok(bRet == FALSE, "InitOnceBeginInitialize should fail\n");
```

Comment:
```suggestion
    ok(!bRet, "InitOnceBeginInitialize should fail\n");
```

Reply:
You are nitpicking.

Reply:
Sure he does. But "short is good"(TM). I see nothing wrong with aiming for excellence in that regard.

Reply:
No, that is complete nonsense.